### PR TITLE
refactor(tooltip): no longer use capture events.

### PR DESCRIPTION
**Pull request by driskull**
**Related Issue:** None

## Summary

- Update `TooltipManager` to no longer use capture events.
- preventDefault() on escape key pressed from a referenceElement or active tooltip.

### CODE REVIEW

File: packages/calcite-components/src/components/tooltip/TooltipManager.ts [hunk 1]
```diff
@@ -79,10 +79,13 @@ export default class TooltipManager {
       if (activeTooltip?.open) {
         this.clearHoverTimeout();
         this.closeActiveTooltip();
-
         const referenceElement = getEffectiveReferenceElement(activeTooltip);
+        const composedPath = event.composedPath();
 
-        if (referenceElement instanceof Element && referenceElement.contains(event.target as HTMLElement)) {
+        if (
+          (referenceElement instanceof Element && composedPath.includes(referenceElement)) ||
```
Comment (driskull) [inline packages/calcite-components/src/components/tooltip/TooltipManager.ts:86]: preventDefault on escape key pressed from a referenceElement or active tooltip.

File: packages/calcite-components/src/components/tooltip/tooltip.e2e.ts [hunk 2]
```diff
@@ -602,7 +612,7 @@ describe("calcite-tooltip", () => {
     expect(await hoverTip.getProperty("open")).toBe(false);
 
     await page.$eval("#hoverRef", (el: HTMLElement) => {
-      el.dispatchEvent(new PointerEvent("pointermove"));
+      el.dispatchEvent(new PointerEvent("pointermove", { cancelable: true, bubbles: true }));
```
Comment (jcfranco) [inline packages/calcite-components/src/components/tooltip/tooltip.e2e.ts:615]: Nitpick: common event options could be DRY'd up. ☀️

**Review by jcfranco — APPROVED** [review]
Awesome!

Can we make sure to land this after our August (R3) release?

**Comment by driskull** [comment]
> Can we make sure to land this after our August (R3) release?

@jcfranco can we land this in the July release?

**Comment by jcfranco** [comment]
I'd like to hold off until September to prevent teams encountering issues similar to #9013 near the R3 release.

**Comment by driskull** [comment]
Ok sounds good

**Comment by jcfranco** [comment]
> I'd like to hold off until September to prevent teams encountering issues similar to https://github.com/Esri/calcite-design-system/issues/9013 near the R3 release.

Belated apologies, I meant to say "hold off until October or at least after September" (post-R3). I'll adjust the milestone.

**Review by jcfranco — APPROVED** [review]

**Pull request merged**